sdks/java/core/src/test/java/org/apache/beam/sdk/coders/CoderFactoriesTest.java

```diff
@@ -38,48 +42,53 @@
    * exercise CoderFactoryFromStaticMethods.
    */
   @Test
-  public void testAtomicCoderClassFactories() {
-    checkAtomicCoderFactory(StringUtf8Coder.class, StringUtf8Coder.of());
-    checkAtomicCoderFactory(DoubleCoder.class, DoubleCoder.of());
-    checkAtomicCoderFactory(ByteArrayCoder.class, ByteArrayCoder.of());
+  public void testAtomicCoderClassFactories() throws Exception {
+    checkAtomicCoderFactory(String.class, StringUtf8Coder.class, StringUtf8Coder.of());
```

Comment:
What is being checked? This test is not readable. Not your fault, but the method adds nothing; I'd inline it.